src/components/PvGenerator/PvGeneratorComponent.tsx

```diff
@@ -19,6 +19,9 @@ export enum PvGeneratorComponentTabs {
 @observer
 export class PvGeneratorComponent extends React.Component<WidgetProps> {
     @observable selectedTabId: TabId = PvGeneratorComponentTabs.PV_IMAGE;
+    axesOrder = {};
+    @observable isConfirmationDialogOpen: boolean = false;
+    @observable error: boolean = false;
```

Comment:
perhaps rename to `isValidSpectralRange`

Reply:
Updated

Reply:
Sorry, looks like the meaning is opposite now. perhaps rename to `inValidSpectralRange`

Reply:
I think inValidSpectralRange would mean the same thing as isValidSpectralRange. I might as well change the logic that uses isValidSpectralRange and keep the name?